libsawtooth/src/journal/validation_rule_enforcer.rs

```diff
@@ -15,14 +15,16 @@
  * ------------------------------------------------------------------------------
  */
 
-use sawtooth::{batch::Batch, state::settings_view::SettingsView, transaction::Transaction};
+use crate::{batch::Batch, state::settings_view::SettingsView, transaction::Transaction};
 
 /// Retrieve the validation rules stored in state and check that the
 /// given batches do not violate any of those rules. These rules include:
 ///
+/// ```ignore
```

Comment:
Why was this added?

Reply:
This piece was causing the docs test to fail because it seemed to be incorrectly parsing the line (it was throwing an error for N not being followed by the correct symbol)

Reply:
Can you add a comment to the commit. I had a simular issues (but it already had the `). I think its caused by the difference between 2015 and 2018 rust.

Reply:
Ok, yeah definitely.